app/views/Main.js

```diff
@@ -7,82 +7,78 @@ import Colors from '../constants/colors';
 import { isGPS } from '../COVIDSafePathsConfig';
 import { checkIntersect } from '../helpers/Intersect';
 import BackgroundTaskServices from '../services/BackgroundTaskService';
-import LocationServices, { Reason } from '../services/LocationService';
-import { ExposureNotificationNotAvailablePage } from './main/ExposureNotificationNotAvailablePage';
+import LocationServices from '../services/LocationService';
+import ExposureNotificationService from '../services/ExposureNotificationService';
```

Comment:
I wouldnt follow the LocationService as a guide here, i dont think they we ultimately be very similar work flows.  Its not clear to me what the ExposureNotificationService would even do as most of the EN interaction is happing in the native layer?

Reply:
We should chat about how you want to use the ExposureNotificationContext instead. I was trying to minimize the isGPS logic and use dependency injection.

Reply:
got it, that makes sense to me as a strategy, I wonder if we could even do this at a higher level in the tree than the MainScreen, feels like this wants to be during the app initialization step.